js_modules/dagster-ui/packages/ui-core/src/runs/AssetTagCollections.tsx

```diff
@@ -165,7 +183,7 @@ export const AssetKeyTagCollection = React.memo((props: AssetKeyTagCollectionPro
 
   const {containerRef, moreLabelRef} = useAdjustChildVisibilityToFill(moreLabelFn);
 
-  if (!assetKeys || !assetKeys.length) {
+  if (!sortedAssetKeys || !sortedAssetKeys.length) {
```

Comment:
```suggestion
  if (!count) {
```

Reply:
https://github.com/dagster-io/dagster/pull/26474/files#diff-adee9343e5e2063f6420a3cae97a413c1f43a9b9cb8241fd9866cc0c45f628bdR186

https://github.com/dagster-io/dagster/pull/26474/files#diff-adee9343e5e2063f6420a3cae97a413c1f43a9b9cb8241fd9866cc0c45f628bdR314

We still need the !assetKeys || !count cuz assetKeys/assetChecks can be null

Reply:
Oh wait nvm I see the code for count I'm revert to just count